test-suite/Makefile

```diff
@@ -119,6 +117,9 @@ PREREQUISITELOG = prerequisite/admit.v.log			\
 all: run
 	$(MAKE) report
 
+# do nothing
+noop: ;
```

Comment:
Why do we need this noop thing?
Shouldn't it be phony?

Reply:
It should be phony indeed.

I introduced the noop as to avoid dealing with empty env vars, but maybe it can be avoided, let me see.

Reply:
Well, the thing is that I do this:
```
   (run make -j 2 BIN= PRINT_LOGS=1 UNIT_TESTS=%{env:COQ_UNIT_TEST=unit-tests}))))
```
so if `COQ_UNIT_TEST` is not set, I enable them. So I pass `noop` to disable it. This is not so bad, as the other choice would be to disable by default, the point is that it is hard to unset a make variable if an envar is set.